porebuilder/examples/func_dry_pore.py

```diff
@@ -0,0 +1,46 @@
+from porebuilder import GraphenePoreFunctionalized
+import mbuild as mb
+
+
+class H(mb.Compound):
+    def __init__(self):
+        super(H, self).__init__()
+
+        self.add(mb.Particle(name='H'))
+        up_port = mb.Port(
+            anchor=self[0], orientation=[
+                0, 1, 0], separation=.075)
+        self.add(up_port, "up")
+
+
+class O(mb.Compound):
+    def __init__(self):
+        super(O, self).__init__()
+
+        self.add(mb.Particle(name='O'))
+        up_port = mb.Port(
+            anchor=self[0], orientation=[
+                0, 1, 0], separation=.075)
+        self.add(up_port, "up")
```

Comment:
I think this should be named `down`. It's not a strict requirement but it's a convention from `mb.Polymer` that I would like to continue for simple systems.